test/service/test_manifest.py

```diff
@@ -1087,6 +1087,41 @@ def test_manifest_content_disposition_header(self, _can_use_cached_manifest):
                             actual_cd = one(parse_qs(query).get('response-content-disposition'))
                             self.assertEqual(actual_cd, expected_cd)
 
+    @mock_sts
+    @mock_s3
+    def test_full_hash_validity(self):
+        self.maxDiff = None
+        old_bundle = ("aaa96233-bf27-44c7-82df-b4dc15ad4d9d", "2018-11-02T113344.698028Z")
+        self._index_canned_bundle(old_bundle)
+        # When a new bundle is indexed and its full manifest cached,
```

Comment:
While I like the explanatory comments, the empty lines are in the wrong places it seems. Usually, I'll put an empty line before every comment.